recipe/meta.yaml

```diff
@@ -1,25 +1,25 @@
-{% set version = "0.6" %}
+{% set version = "0.8" %}
 
 package:
   name: terminado
   version: {{ version }}
 
 source:
-  fn: terminado-{{ version }}.tar.gz
-  url: https://github.com/takluyver/terminado/archive/0.6.tar.gz
-  sha256: 6a1cfe8ede41962827ca8260bc64f387de85be62fd4e936bdde2ecf2a433931c
+  url: https://pypi.io/packages/source/t/terminado/terminado-{{ version }}.tar.gz
+  sha256: 984c914303fcd1642c1783cdc6727db24b0bc1494fa6363e6ae46696335e2944
 
 build:
   number: 0
   script: python setup.py install
```

Comment:
One thing I was wondering though is how this works if there's no `setup.py` file in the repo??

I thought perhaps it required flit now? I don't know that project well but had changed my script to:
```
script: pip install flit && flit install
```
...to work around the fact there appears to be no conda package for flit.

I guess that's redundant if the usual `python setup.py install` still works?

Reply:
Terminado doesn't have a `setup.py` at all, it uses the `pyproject.toml` file from https://www.python.org/dev/peps/pep-0518/.

Reply:
...but the build command in the `meta.yaml`  is `python setup.py install` - that can't work right?

Reply:
...well, it does seem to have worked so I'll have to look into that!

Reply:
The release tarballs on PyPI include a generated `setup.py`, so that will work.

It would be a good idea to move over to `pip install --no-deps .`, though: once pip is compatible with PEP 517, the setup.py won't be needed any more.